search/src/test/scala/weco/api/search/works/WorksFiltersTest.scala

```diff
@@ -908,94 +752,36 @@ class WorksFiltersTest
     it("filters by multiple comma-separated availability IDs") {
       withWorksApi {
         case (worksIndex, routes) =>
-          insertIntoElasticsearch(
-            worksIndex,
-            works: _*
-          )
-          assertJsonResponse(
-            routes = routes,
-            unordered = true,
-            path = s"$rootPath/works?availabilities=closed-stores,online"
-          ) {
-            Status.OK -> worksListResponse(
-              works =
-                Seq(onlineWork, closedStoresWork, onlineAndClosedStoresWork)
+          indexTestDocuments(worksIndex, worksEverything ++ visibleWorks: _*)
+
+          assertJsonResponse(routes, path = s"$rootPath/works?availabilities=open-shelves,closed-stores") {
+            Status.OK -> newWorksListResponse(
+              ids = Seq("work.visible.everything.0", "work.visible.everything.1", "work.visible.everything.2")
             )
           }
       }
     }
   }
 
   describe("relation filters") {
-
-    def work(path: String): Work.Visible[Indexed] =
-      indexedWork(sourceIdentifier = createSourceIdentifierWith(value = path))
-        .collectionPath(CollectionPath(path = path))
-        .title(s"The title is $path")
-
-    val workA = work("A")
-    val workB = work("A/B").ancestors(workA)
-    val workC = work("A/C").ancestors(workA)
-    val workD = work("A/C/D").ancestors(workA, workC)
-    val workE = work("A/C/D/E").ancestors(workA, workC, workD)
-    val workX = work("X")
-
-    def storeWorks(index: Index) =
-      insertIntoElasticsearch(index, workA, workB, workC, workD, workE, workX)
-
-    it("filters partOf by id from root position") {
+    it("filters partOf by id") {
       withWorksApi {
         case (worksIndex, routes) =>
-          storeWorks(worksIndex)
-          assertJsonResponse(routes, s"$rootPath/works?partOf=${workA.id}") {
-            Status.OK -> worksListResponse(
-              works =
-                Seq(workB, workC, workD, workE).sortBy(_.state.canonicalId)
-            )
-          }
-      }
-    }
+          indexTestDocuments(worksIndex, worksEverything: _*)
 
-    it("filters partOf by title from root position") {
```

Comment:
I simplified these tests because really they're a property of the indexing, not the API itself.